app/javascript/packs/hello_react.jsx

```diff
@@ -2,25 +2,23 @@
 // like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
 // of the page.
 
-import React from 'react'
-import ReactDOM from 'react-dom'
-import PropTypes from 'prop-types'
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
 
-const Hello = props => (
-  <div>Hello {props.name}!</div>
-)
+const Hello = props => <div>Hello {props.name}!</div>;
 
 Hello.defaultProps = {
-  name: 'David'
-}
+  name: "David",
+};
 
 Hello.propTypes = {
-  name: PropTypes.string
-}
+  name: PropTypes.string,
+};
 
-document.addEventListener('DOMContentLoaded', () => {
+document.addEventListener("DOMContentLoaded", () => {
   ReactDOM.render(
     <Hello name="React" />,
-    document.body.appendChild(document.createElement('div')),
-  )
-})
+    document.body.appendChild(document.createElement("div"))
+  );
+});
```

Comment:
I had modified this file to test pre-commit hook. I have checked in the modified file which Prettier formatted according to Eslint rules. I guess this is fine.